test-manager/Cargo.toml

```diff
@@ -29,11 +29,11 @@ env_logger = "0.9"
 tonic = "0.8"
 tower = "0.4"
 colored = "2.0.0"
-mullvad-management-interface = { git = "https://github.com/mullvad/mullvadvpn-app" }
-old-mullvad-management-interface = { git = "https://github.com/mullvad/mullvadvpn-app", rev = "2022.5", package = "mullvad-management-interface" }
-talpid-types = { git = "https://github.com/mullvad/mullvadvpn-app" }
-mullvad-types = { git = "https://github.com/mullvad/mullvadvpn-app" }
-mullvad-api = { git = "https://github.com/mullvad/mullvadvpn-app" }
+mullvad-management-interface = { git = "https://github.com/mullvad/mullvadvpn-app", branch = "main" }
+old-mullvad-management-interface = { git = "https://github.com/mullvad/mullvadvpn-app", rev = "2023.2", package = "mullvad-management-interface" }
```

Comment:
Is there a way we can avoid having to hardcode the stable version? You mentioned using the CLI instead. This is not a problem for this PR.

Reply:
Yes, we will probably want to use some other method since we might want to test against many versions.